Solutions/vArmour/Analytic Rules/vArmourApplicationControllerSMBRealmTraversal.yaml

```diff
@@ -0,0 +1,22 @@
+id: a36de6c3-3198-4d37-92ae-e19e36712c2e
+name: vArmour AppController - SMB Realm Traversal
+description: |
+  'Detects when SMB traffic crosses Production and Non-Production Realms.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: vArmourAC
+    dataTypes:
+      - "CommonSecurityLog (vArmour)"
```

Comment:
Just "CommonSecurityLog" is enough for dataTypes, please remove "(vArmour)".

Reply:
Updated as requried.